driver/ixsystems/freenasapi.py

```diff
@@ -181,6 +182,7 @@ def _get_error_info(self, err):
         self.COMMAND_RESPONSE['status'] = self.STATUS_ERROR
         if isinstance(err, urllib.error.HTTPError):
             self.COMMAND_RESPONSE['response'] = '%d:%s' % (err.code, err.msg)
+            self.COMMAND_RESPONSE['code'] = '%d' % err.code
```

Comment:
Why string and not integer?

Reply:
I might be wrong, please comments, I was thinking str is more safe than integer, and given a default value '' for response.code value, if we put it as integer what default value is better for response.code field?

Reply:
I dont understand why you need a default. But if you do seems like -1 could do the job just fine.